js/apps/shared/constants.js

```diff
@@ -0,0 +1,13 @@
+(() => {
+	'use strict';
+
+	const app = angular.module('sharedConstants', []);
+
+	const constants = {
+		urls: {
+			getBranches: 'data/branch-amenities.json'
```

Comment:
Is this the URL in staging too, or will this break?

Reply:
not being used, removing